### PR TITLE
Add Maintenance actions to bare metal node

### PR DESCRIPTION
**Pull request by jtomasek**
~~Depends on https://github.com/openshift/console/pull/2969~~
Required by https://github.com/openshift/console/pull/3066

### CODE REVIEW

File: frontend/packages/metal3-plugin/src/components/baremetal-nodes/menu-actions.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { Kebab, KebabOption, asAccessReview } from '@console/internal/components/utils';
+import {
+  MarkAsSchedulable,
+  MarkAsUnschedulable,
+} from '@console/app/src/components/nodes/menu-actions';
+import { K8sKind, NodeKind, K8sResourceKind } from '@console/internal/module/k8s';
+import { getName } from '@console/shared';
+import { startNodeMaintenanceModal } from '../modals/StartNodeMaintenanceModal';
+import stopNodeMaintenanceModal from '../modals/StopNodeMaintenanceModal';
```
Comment (vojtechszocs) [inline frontend/packages/metal3-plugin/src/components/baremetal-nodes/menu-actions.ts]: Nit: maybe improve consistency e.g. use named or default exports for all modal components :smiley: can be done later.

Comment (jtomasek) [reply frontend/packages/metal3-plugin/src/components/baremetal-nodes/menu-actions.ts]: oops 😃, I'll fix this in subsequent PR.

**Pull request merged**